Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -1,127 +1,205 @@
-      :file:`Classes/`. See :ref:`extension files locations <extension-files-locations>`
-      for more information on how single extensions are structured.
-
-  - :Directory: :ref:`Environment-config-path` either :file:`typo3conf/` or :file:`config/`
-    :Description:
-      TYPO3 configuration directory. This directory contains installation wide
-      configuration.
-
-      The most important file within this folder is
-      :file:`LocalConfiguration.php`. This one contains local settings of the
-      main global PHP array :php:`$GLOBALS['TYPO3_CONF_VARS`], crucial settings
-      like database connect credentials are in here. The file is managed by the
-      Install Tool and the Extension Manager and the content should not be
-      managed manually since Extension Manager or Install Tool may override
-      manually changed settings again.
-
-      The file :file:`LocalConfiguration.php` can be enriched by
-      :file:`AdditionalConfiguration.php` which is never touched by TYPO3
-      internal management tools. Be aware that having settings within
-      :file:`AdditionalConfiguration.php` may prevent the system from doing
-      automatic upgrades and should be used with care and only if you know what
-      you are doing.
-
-  - :Directory: :file:`typo3conf/ext/`
-    :Description:
-      Directory for local TYPO3 extensions. Each subdirectory contains one extension.
-
-  - :Directory: :ref:`Environment-labels-path` either :file:`typo3conf/l10n` or :file:`var/labels`
-    :Description:
-      Directory for extension localisations. Contains all downloaded translation
-      files.
-
-  - :Directory: :file:`typo3temp/`
-    :Description:
-      Directory for temporary files. It contains subdirectories (see below)
-      for temporary files of extensions and TYPO3 components.
-
-  - :Directory: :file:`typo3temp/assets/`
-    :Description:
-      Directory for temporary files that should be public available
-      (e.g. generated images).
-
-  - :Directory: :file:`typo3temp/var/`
-    :Description:
-      Directory for temporary files that contains private files (e.g. cached
-      Fluid templates) and should not be publicly available.
-      See also :ref:`Environment-configuring-paths` for a more detailed description.
+.. _directory-project:
+
+Files on project level
+======================
+
+On the top most level, the project level, you can find the files
+:file:`composer.json` which contains requirements for the TYPO3 installation
+and the :file:`composer.lock` which contains information about the concretely
+installed versions of each package.
+
+.. _directory-config:
+
+:file:`config`
+==============
+
+TYPO3 configuration directory. This directory contains installation wide
+configuration.
+
+.. _directory-config-sites:
+
+:file:`sites/`
+--------------
+
+The folder :file:`config/sites` contains subfolders for each
+:ref:`site configuration <sitehandling>`.
+
+.. _directory-local_packages:
+
+:file:`local_packages` (optional)
```

Comment:
Here we use "local_packages": https://docs.typo3.org/m/typo3/tutorial-sitepackage/main/en-us/ExtensionInstallation/Index.html

I kind of prefere this name because it makes clear that it is a local package not one installed from somewhere else, but I agree, we should call it the same all throughout the documentation